CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.81.0](https://github.com/rudderlabs/rudder-transformer/compare/v1.79.1...v1.81.0) (2024-10-04)
+
+
```

Comment:
@utsabc duplicate changelog of 1.80.0, need to be cleaned

Reply:
Is the process 
- After merging to develop cleaning it?
- Or use some admin privileges to clean it directly in main?

Reply:
Ideally in the hotfix release branch, it has to be cleaned up.
Now, we can do it after merging to develop & later on it gets propagated to main as part of next release

Reply:
ok